Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
     encryptor (3.0.0)
     erubi (1.12.0)
     erubis (2.7.0)
-    execjs (2.7.0)
-    faraday (2.7.11)
+    execjs (2.9.1)
+    faraday (2.8.1)
       base64
       faraday-net_http (>= 2.0, < 3.1)
       ruby2_keywords (>= 0.0.4)
-    faraday-http-cache (2.5.0)
+    faraday-http-cache (2.5.1)
       faraday (>= 0.8)
     faraday-multipart (1.0.4)
       multipart-post (~> 2)
     faraday-net_http (3.0.2)
     faraday-net_http_persistent (2.1.0)
       faraday (~> 2.5)
       net-http-persistent (~> 4.0)
-    ffi (1.15.5)
+    ffi (1.16.3)
     ffi-compiler (1.0.1)
       ffi (>= 1.0.0)
       rake
-    flay (2.12.0)
+    flay (2.12.1)
       erubis (~> 2.7.0)
       path_expander (~> 1.0)
       ruby_parser (~> 3.0)
       sexp_processor (~> 4.0)
     flowdock (0.7.1)
       httparty (~> 0.7)
       multi_json
-    forking_test_runner (1.2.0)
+    forking_test_runner (1.13.0)
```

Comment:
### [forking_test_runner](https://github.com/grosser/forking_test_runner)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.2.0 → 1.13.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/forking_test_runner/1.2.0/1.13.0)_]





<details>
<summary>Commits</summary>

A change of **48** commits. See the full changes on [the compare page](https://github.com/grosser/forking_test_runner/compare/v1.2.0...v1.13.0).

These are the first 10 commits:
- (1ecb4e3) [load version so it is available for --version (never needed …](https://github.com/grosser/forking_test_runner/commit/1ecb4e335757e3e0808953b3621e7c05db93c937)
- (864d0d7) [allow to disable AR connection management](https://github.com/grosser/forking_test_runner/commit/864d0d784ed31a59919421aeb64193ba1b1a5800)
- (b7f5b89) [v1.3.0](https://github.com/grosser/forking_test_runner/commit/b7f5b892fcf4cbb75cbb952a3f4be29321433032)
- (9732c56) [Add support for Buildkite environment](https://github.com/grosser/forking_test_runner/commit/9732c56b1895b20105b2950c26d2f61ae3891124)
- (c3827a8) [Fix typo](https://github.com/grosser/forking_test_runner/commit/c3827a8987fdc6c7b0789f300b78e58bc3d092b2)
- (7120c43) [Merge pull request #12 from buildkite/add-buildkite-support](https://github.com/grosser/forking_test_runner/commit/7120c4347310ad04c1663b2156d1ad62eaeaa10c)
- (b637d00) [fix buildkite additions](https://github.com/grosser/forking_test_runner/commit/b637d0087c8c25a3e837e795c3d1c367b70d62a9)
- (4dd8fe1) [v1.4.0](https://github.com/grosser/forking_test_runner/commit/4dd8fe183d51cbf5737c8a7c9423795cbfddb4dd)
- (6d837fd) [bump](https://github.com/grosser/forking_test_runner/commit/6d837fd055975ad884b024676af6ea89b4b703f3)
- (9a57cfc) [cleanup](https://github.com/grosser/forking_test_runner/commit/9a57cfc6b061c3b80e4b8c88ab5a7678d4435317)


</details>